GoogleChromeLabs/ps-analysis-tool
# Remove `reCaptcha` from library detection

## Description
This PR handles removal of reCaptcha from library detection package.
- Remove the callbacks to reCaptcha files.
- Remove files containing code related to reCaptcha detection.


## Relevant Technical Choices



## Testing Instructions
- Navigate to site https://online.1c.ru/
- Open PSAT panel and navigate to cookies landing page.
- Move to known breakages section.
- The section should be empty with no libraries detected text.
- Now do same steps with `develop`branch/0.70v and compare.



## Additional Information:



## Screenshot/Screencast
<img width="1470" alt="Screenshot 2024-05-08 at 11 29 38" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/d89e39f9-6a24-4a3d-b060-b82a274ed276">



---

## Checklist



- ~This code is covered by unit tests to verify that it works as intended.~



Fixes #
